packages/testing-docs/docs/BE Tests/ Common /Networks navigation - Staging/id562 - Networks navigation Staging - Common - Navigation to zkSync Era Testnet.md

```diff
@@ -0,0 +1,17 @@
+---
+tags: ['Full test', 'manual', 'Navigation', 'Network', 'Smoke test', 'ZKF-2507', 'Automated']
```

Comment:
Finally, can you please sort/select/take attention is the test case 'manual' or 'automated'? We have to keep only one of this tags.

Reply:
Yes, fixed.
Thank you!